stable/wordpress/values.yaml

```diff
@@ -41,7 +41,7 @@ wordpressBlogName: User's Blog!
 
 ## Set to `yes` to allow the container to be started with blank passwords
 ## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
-allowEmptyPassword: yes
+allowEmptyPassword: true
```

Comment:
we should leave this as-is

Reply:
is there any reason?

Reply:
This environment variables is used by the WP [Nami](https://github.com/bitnami/nami) module which won't recognised "true" as a valid value.

Valid values are: ["yes", "no"]